doc/src/modules/physics/mechanics/examples/four_bar_linkage_example.rst

```diff
@@ -0,0 +1,61 @@
+==================
+A four bar linkage
+==================
+
+The four bar linkage is a common example used in mechanics, which has to holonomic constraints. This example will make use of joints functionality provided in :mod:`sympy.physics.mechanics`. In summary we will use bodies and joints to define the open loop system. Next, we define the configuration constraints to close the loop. With the ``JointsMethod`` we get the input to manually setup the ``KanesMethod`` as its backend.
+
+.. image:: four_bar_linkage.svg
+   :align: center
```

Comment:
Probably a good idea to set a width for svgs.

Reply:
Thanks for the suggestion.